cfpb/consumerfinance.gov
# Add 1071 Filing Instruction Guide MVP

Add a new wagtail page type that includes a table of contents.  This will be the basis of the new filing instruction guide for 1071. 

---




## Additions

- Models, templates, and migrations for the FigContentPage. 


## How to test this PR

1. Run this branch and create a filing instruction guide page.  Make sure that shows sections, subsections, and sub-subsections on the left hand side. 


## Notes and todos

- This is a work in progress on the 1071 FIG team and is the basic MVP that we, @schbetsy  and I, will be working on overtime. 


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
